[IMP] Spreadsheets: updating image and doc links (#9522)
odoo/documentation: content/applications/productivity/spreadsheet.rst

   spreadsheet/functions
   spreadsheet/templates

> Spreadsheets allow users the ability to organize, analyze, and visualize data in a tabular form.
> Various functions that can be performed within a spreadsheet are:

Reviewer: ```suggestion
Spreadsheets allow users to organize, analyze, and visualize data by compiling data sets into tables.
Various functions that can be performed within a spreadsheet are:
```

Reviewer: Optional rewording for clarity!

> - Insert and :doc:`link Odoo data <spreadsheet/insert>` (pivots, graphs, lists, and menus) .

Reviewer: ```suggestion
- Insert and :doc:`link Odoo data <spreadsheet/insert>` (pivots, graphs, lists, and menus).
```

   spreadsheet/functions
   spreadsheet/templates

> Spreadsheets allow users to organize, analyze, and visualize data by compiling data sets into

Reviewer 2: ```suggestion
Spreadsheets help organize, analyze, and visualize data by compiling data sets into
```
